content/docs/operations/sample-app.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Sample app
+weight: 7
+description: Deploy httpbin as a sample app.
+---
+
+The httpbin app lets you test your API gateway by sending requests to it and receiving responses.
+
+## Before you begin
+
+Set up {{< reuse "docs/snippets/product-name.md" >}} by following the [Quick start](/docs/quickstart/) or [Installation](/docs/operations/install/) guides.
+
+## Deploy a sample app {#deploy-app}
+
+The following configuration file creates the httpbin app. To review the source file, see [the kgateway project](https://github.com/kgateway-dev/kgateway/blob/main/examples/httpbin.yaml).
+
+1. Create the httpbin app.
+
+   ```shell
+   kubectl apply -f https://raw.githubusercontent.com/kgateway-dev/kgateway/refs/heads/main/examples/httpbin.yaml
+   ```
+
+   Example output:
+   
+   ```txt
+   namespace/httpbin created
+   serviceaccount/httpbin created
+   service/httpbin created
+   deployment.apps/httpbin created
+   ```
+
+2. Verify that the httpbin app is running.
+   
+   ```sh
+   kubectl -n httpbin get pods
+   ```
+
+   Example output: 
+   
+   ```txt
+   NAME                      READY   STATUS    RESTARTS   AGE
+   httpbin-d57c95548-nz98t   3/3     Running   0          18s
+   ```
+
+## Set up an API gateway {#api-gateway}
+
+Create an API gateway with an HTTP listener by using the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}}.
+
+1. Create a gateway resource and configure an HTTP listener. The following gateway can serve HTTP resources from all namespaces.  
+   
+   ```yaml
+   kubectl apply -f- <<EOF
+   kind: Gateway
+   apiVersion: gateway.networking.k8s.io/v1
+   metadata:
+     name: http
+     namespace: {{< reuse "docs/snippets/ns-system.md" >}}
+   spec:
+     gatewayClassName: kgateway
+     listeners:
+     - protocol: HTTP
+       port: 8080
+       name: http
+       allowedRoutes:
+         namespaces:
+           from: All
+   EOF
+   ```
+
+2. Verify that the gateway is created successfully. You can also review the external address that is assigned to the gateway. Note that depending on your environment it might take a few minutes for the load balancer service to be assigned an external address. If you are using a local Kind cluster without a load balancer such as `metallb`, you might not have an external address.
```

Comment:
```suggestion
2. Verify that the Gateway is created successfully. You can also review the external address that is assigned to the Gateway. Note that depending on your environment it might take a few minutes for the load balancer service to be assigned an external address. If you are using a local Kind cluster without a load balancer such as `metallb`, you might not have an external address.
```